egs2/chime7_task1/asr1/local/install_dependencies.sh

```diff
@@ -23,10 +23,10 @@ ${MAIN_ROOT}/tools/installers/install_s3prl.sh
 
 if ! command -v gss &>/dev/null; then
   conda install -yc conda-forge cupy=10.2
-  ${MAIN_ROOT}/tools/installers/install_gss.sh.
+  ${MAIN_ROOT}/tools/installers/install_gss.sh
 fi
 
-sox_conda=`command -v ../../../tools/venv/bin/sox 2>/dev/null`
+sox_conda=`command -v $(dirname $(which python))/sox 2>/dev/null`
```

Comment:
hopefully this fixes the sox issue.

Reply:
FYI. Conda setups some useful shell environment variables: CONDA_PREFIX, CONDA_EXE, etc.

If sox was installed by sox, the path should be ${CONDA_PREFIX}/bin/sox

Reply:
Many thanks I was not aware of CONDA_PREFIX. Seems much better to use that, it is more clean

Reply:
Updated. Also updated [local/generate_chime6_data.sh] with same variable (https://github.com/espnet/espnet/pull/4934/files#diff-83abe1d71c47a59774fbbdc7721c39519004e681d12764a55c81c66f9ffaceae)

Reply:
@kamo-naoyuki I followed your suggestion and added a script + JSON file to check the MD5 checksum for each file https://github.com/espnet/espnet/blob/cfbb957d9c71c5c7aed27a1d4b2b85b62721381a/egs2/chime7_task1/asr1/local/check_data_gen.py but I had also to add a .json file to this recipe. Is it ok ?